server/public/model/user.go

```diff
@@ -105,6 +105,7 @@ type User struct {
 	TermsOfServiceId       string    `json:"terms_of_service_id,omitempty"`
 	TermsOfServiceCreateAt int64     `json:"terms_of_service_create_at,omitempty"`
 	DisableWelcomeEmail    bool      `json:"disable_welcome_email"`
+	LastLogin              int64     `json:"last_login,omitempty"`
```

Comment:
I don't see anything sanitizing this value, so I guess it is sent with any other request that fetch a user. Am I right? If so, is it something we want to broadcast that way?

Reply:
Yeah, good catch. This doesn't seem like something that should be shown to anyone outside of admins.
Where should this be sanitized?

Reply:
I am almost sure there is a "SanitizeUser" function, or a "Sanitize" method in the user object (yeah yeah, I know that is no Go vocabulary, but you get me). In theory, you can track it down in whatever API call that sends users back to the client.